pages/templates/pages/app/sequences/index.html

```diff
@@ -10,21 +10,37 @@ <h2>{{ sequence.title }}</h2>
                 <ul>
                     {% for element in elements %}
                     <li>
-                        <a href="{% url 'sequences:sequence_page_element_view' sequence=element.sequence.slug rank=element.rank %}">
+                        {% if element.is_live_event %}
+                            <span class="live-event-label">
+                                <a href="{% url 'sequences:sequence_page_element_view' sequence=element.sequence.slug rank=element.rank %}">
```

Comment:
Typically we pass the URLs in an `urls` variable through the `context` such as to make the templates Jinja2 compatible.